blockchain/README.md

```diff
@@ -0,0 +1,64 @@
+## Prerequisits (one time)
+
+# Requirements
+
+- Windows 10 with 64 bit 
+- Windows 11 with 64 bit operating system 
+
+# How to install on windows 
+
+- The first place to start is the official Docker website(https://www.docker.com/get-started/) from where  download Docker Desktop.
+- Run the downloaded installer 
+- Configuration
+    Mark both options true
+- Click Ok and wait a bit
+- Restart required
+- Docker automatically run Accept License Agreement
+- the Docker Desktop window will open. we have selected WSL 2 as our virtualization enginein configuration, we also need to install it. Don’t click Restart just yet!
+- Follow the link in the dialog window and download WSL 2.
+- Open the installer.
+- Click on Next to begin installing the Windows Subsystem for Linux (WSL).
+- Restart required
+
+- reference: https://www.linkedin.com/pulse/step-guide-how-install-docker-windows-1011-shashank-abhishek/
+
+## Create a new keypair
+This script is used to create a new key pair in the Antelope blockchain.
+
+- docker run -it tonomy_blockchain:easycleos ./create-key.sh
+
+After running the script, it will generate a private key. Store the private key in a very safe place. It will be required for the transact.sh script.
+
+## Sign blockchain transactions
+This script is used to make transactions in the Antelope blockchain.
+
+- docker run -it tonomy_blockchain:easycleos ./transact.sh
```

Comment:
use a 
```bash
# bash snippet
docker run -it tonomy_blockchain:easycleos ./transact.sh
```
or code quotes
`docker run -it tonomy_blockchain:easycleos ./transact.sh`
to show this is the code to put in